modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,314 @@
+min-width and max-width.
+
+> **Note**: This function subtracts `0.5px` from the next breakpoint value to prevent collisions.
+> For example, `breakpoints.values.s`, has a `min-width` of `320px`, and the `max-width` is
+> `767.5px`).
+
+If the `xl` breakpoint is provided, this function returns a media query with only a `min-width`, as
+seen in the second example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {between} = theme.canvas.breakpoints;
+// Returns '@media (min-width: 320px) and (max-width: 1023.5px)'
+const mediaQuery = between('s', 'm');
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+This example uses `xl` as the `end` breakpoint and only adds a min-width to the media query.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {between} = theme.canvas.breakpoints;
+const mediaQuery = between('m', 'xl'); // Returns '@media (min-width: 768px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+`only()` - Returns a media query with a `min-width` and `max-width` for a given breakpoint. Given a
+breakpoint key ("zero", "s", "m", "l", "xl"), this function returns a media query (string) using a
+`min-width` and `max-width`.
+
+> **Note**: This function subtracts `0.5px` from the next breakpoint value to prevent collisions.
+> For example, `breakpoints.values.s`, has a `min-width` of `320px`, and the `max-width` is
+> `767.5px`).
+
+If the `xl` breakpoint is provided, this function returns a media query with only a `min-width` of
+`1440px`, as seen in the second example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+const theme = useTheme();
+const {only} = theme.canvas.breakpoints;
+const mediaQuery = only('s'); // Returns '@media (min-width: 320px) and (max-width: 767.5px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+This example uses the `xl` breakpoint and only adds a `min-width` of `1440px` to the media query.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+const theme = useTheme();
+const {only} = theme.canvas.breakpoints;
+const mediaQuery = only('xl'); // Returns '@media (min-width: 1440px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+### Viewport vs Container
+
+If you are measuring a user's viewport size, then you should be using Viewport Based Responsive
+Styling.
+
+If you only want to measure the size of a container, then you should be using Container Based
+Responsive Styling.
+
+> **Note**: A container's size can be affected by surrounding containers or the viewport size.
+
+### Canvas Kit Breakpoint Sizes
```

Comment:
Rename this heading to Canvas Breakpoints and also consider moving it further up. This could also go in the `## What is Responsive Styling?` section.
```suggestion
### Canvas Breakpoints
```

Reply:
Moved up and added to the `What is Responsive Styling?` section